doc/nrf/libraries/bluetooth_services/mesh/light_temp_srv.rst

```diff
@@ -70,7 +70,23 @@ The states of the extended model are bound to the states in the Light CTL Temper
 Persistent storage
 ===================
 
-None.
+The Light CTL Temperature Server stores the following information:
+
+* Any changes to the Default Light Temperature/Delta UV and Temperature Range states.
+* The last known Temperature level.
+
+This information is used to reestablish the correct Temperature level when the device powers up.
+
+If :kconfig:option:`CONFIG_BT_SETTINGS` is enabled, the Light CTL Temperature Server stores all its states persistently using a configurable storage delay to stagger storing.
+See :kconfig:option:`CONFIG_BT_MESH_MODEL_SRV_STORE_TIMEOUT`.
+
+The Light Hue Server can use the :ref:`emergency data storage (EMDS) <emds_readme>` together with persistent storage to:
```

Comment:
I just realized... From this line, we are talking about the Light Hue Server. Should it be Light Hue Server or Light CTL Temperature Server?